testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
So I just tried this again yesterday. Even if the root level `grpc/composer.json` is going to pull in `datto/protobuf-php` and with my `stanley-cheung` override, as a user of `grpc/grpc`, you are not going to get the `stanley-cheung` override unless the user also puts that in their own `composer.json`. In other words, as a user of `grpc/grpc`, at the minimum, they still need to `require grpc/grpc` and add the `stanley-cheung/protobuf-php` override. It's not ideal but I can spell it out more in grpc's documentation.